package.json

```diff
@@ -13,10 +13,10 @@
     "check:all": "scripts/run_all_checks.sh",
     "dev": "wrangler dev --env staging",
     "deploy": "wrangler publish",
-    "format": "npm-run-all -c \"format:*\"",
+    "format": "run-p -c \"format:*\"",
```

Comment:
Did you check if running eslint and prettier in parallel works without problems?
(run-p is parallel by default)